constraints.txt

```diff
@@ -258,7 +263,6 @@ ruamel.yaml.clib==0.2.8
 scipy==1.12.0
 seaborn==0.13.2
 SecretStorage==3.3.3
-semsimian==0.2.12
```

Comment:
@hrshdhgd What is the right thing to do to ensure OAK works as before?

Reply:
This is because we made `semsimian` an optional dependency. This is expected. If a user wants `semsimian` asa dependency they'll have to `poetry install --extras semsimian` separately. I don't recall how that would transfer to the ODK setup.

Reply:
Can I just add this to the requirements.txt? Just adding a row `semsimian`? https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full

Reply:
Yes